### PR TITLE
SDK maintenance

### PR DESCRIPTION
**Pull request by acooper4960**

### CODE REVIEW

File: test/SdkUnitTests.js [hunk 1]
```diff
@@ -28,6 +37,20 @@ var RedirectURI = 'https://www.docusign.com/api';
...
 var expiresIn = 3600;
 
+if (privateKey) {
+  var buf;
+  if (typeof Buffer.from === 'function') {
+    // Node 5.10+
+    buf = Buffer.from(privateKey, 'base64'); // Ta-da
+  } else {
+    // older Node versions, now deprecated
+    buf = new Buffer(privateKey, 'base64'); // Ta-da
```
Comment (acooper4960) [inline test/SdkUnitTests.js]: safe-buffer

File: test/SdkUnitTests.js [hunk 2]
```diff
@@ -28,6 +37,20 @@ var RedirectURI = 'https://www.docusign.com/api';
...
+  }
```
Comment (acooper4960) [inline test/SdkUnitTests.js]: using safe-buffer in lieu of legacy Buffer

File: test/SdkUnitTestsWithCallback.js [hunk 3]
```diff
@@ -25,6 +29,20 @@ var RedirectURI = 'https://www.docusign.com/api';
...
 
+if (privateKey) {
+  var buf;
+  if (typeof Buffer.from === 'function') {
+    // Node 5.10+
+    buf = Buffer.from(privateKey, 'base64'); // Ta-da
+  } else {
+    // older Node versions, now deprecated
+    buf = new Buffer(privateKey, 'base64'); // Ta-da
+  }
```
Comment (acooper4960) [inline test/SdkUnitTestsWithCallback.js]: using safe-buffer in lieu of legacy Buffer

File: test/OAuthClientTests.js [hunk 4]
```diff
@@ -7,7 +7,6 @@ const app = express();
 const port = process.env.PORT || 3000;
 const host = process.env.HOST || 'localhost';
 /*
-const integratorKey = 'ae30ea4e-xxxx-xxxx-xxxx-fcb57d2dc4df'; // An IK for a non-mobile docusign account
```
Comment (InbarGazit) [inline test/OAuthClientTests.js]: what about the implicit IK? shouldn't we remove it too?

File: test/OAuthClientTests.js [hunk 5]
```diff
@@ -7,7 +7,6 @@ const app = express();
...
 const integratorKeyImplicit = '68c1711f-xxxx-xxxx-xxxx-b49b4211d831'; // an IK with mobile selected in Docusign admin
 const clientSecret = 'b4dccdbe-xxxx-xxxx-xxxx-b2f0f7448f8f';
```
Comment (InbarGazit) [inline test/OAuthClientTests.js]: same for clientSecret?

Comment (acooper4960) [reply test/OAuthClientTests.js]: @InbarGazit these appear to be scrubbed no?

Comment (InbarGazit) [reply test/OAuthClientTests.js]: yes, but it's inconsistent with the others where you get them from env variables

Comment (acooper4960) [reply test/OAuthClientTests.js]: these are comments, they merely show the guid data type these are.

Comment (acooper4960) [reply test/OAuthClientTests.js]: Updated for consistency

**Comment by acooper4960** [comment]
> need some additional cleanup

@InbarGazit file has been deleted

**Pull request merged**